module4/collections-ref.py

```diff
@@ -0,0 +1,45 @@
+import random
+import string
+
+
+# Generate list if random number of dicts. Return: List of dicts.
+def generate_random_list_of_dicts(number_of_dicts=10, dicts_values=100):
+    if number_of_dicts < 2:
```

Comment:
[NICE TO THINK ABOUT] Two things:
- Maybe unify function for num of dicts in range from 0, not from 2? but it's ok to have 2 for our purpose
- Should we really proceed if number_of_dicts not fit the requirements? maybe raise custom exception? I suppose it will be considered in the next modules, but it's nice to notice for further tasks

Reply:
I thought and tried to implement :)